nilearn/reporting/glm_reporter.py

```diff
@@ -292,14 +296,15 @@ def _check_report_dims(report_size):
         width = int(width)
         height = int(height)
     except ValueError:
-        warnings.warn('Report size has invalid values. '
-                      'Using default 1600x800')
+        warnings.warn(
+            "Report size has invalid values. " "Using default 1600x800"
```

Comment:
```suggestion
            "Report size has invalid values. Using default 1600x800"
```

Reply:
thanks: some of the black reformatting leads to some funny things sometimes